src/dashboard.py

```diff
@@ -116,7 +116,6 @@ def __createLogDatabase(self):
                 self.loggerdb.commit()
     
     def log(self, URL: str = "", IP: str = "", Status: str = "true", Message: str = "") -> bool:
-        pass
```

Comment:
What is the pass then try?

Reply:
To be honest I don't remember I put the `try` over there... but it doesn't seem to do anything to I just removed it haha

Reply:
Yeah good for my review.